internal/configs/import_test.go

```diff
@@ -143,6 +144,29 @@ func TestImportBlock_decode(t *testing.T) {
 			},
 			"Missing required argument",
 		},
+		"error: blank id argument": {
+			&hcl.Block{
+				Type: "import",
+				Body: hcltest.MockBody(&hcl.BodyContent{
+					Attributes: hcl.Attributes{
+						"to": {
+							Name: "to",
+							Expr: bar_expr,
+						},
+						"id": {
+							Name: "id",
+							Expr: blank_str_expr,
+						},
+					},
+				}),
+				DefRange: blockRange,
+			},
+			&Import{
+				ID:        mustAbsResourceInstanceAddr("test_instance.bar"),
```

Comment:
```suggestion
				To:        mustAbsResourceInstanceAddr("test_instance.bar"),
```

or `want` can just be nil here, since it is not checked if `err` is present in the test case.